javascript-modal/styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  box-sizing: border-box;
+}
+
+.open-button {
+  border-radius: 5px;
+  border: none;
+  background-color: rgb(71 71 224);
+  color: white;
+  width: 10em;
+  height: 2em;
+  font-family: sans-serif;
+  font-size: 20px;
+}
+
+h2 {
+  font-family: sans-serif;
+  padding-bottom: 2em;
+}
+
+.no-button {
+  border-radius: 5px;
+  background-color: rgb(230 73 73);
+  color: white;
+  border: none;
+  width: 4em;
+  height: 2em;
+  text-align: center;
+  font-size: 24px;
+}
+
+.modal-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  opacity: 0;
```

Comment:
You shouldn't use `opacity` to hide and show elements, you should make a class with `display: none` as its only styling, and add that class to elements that should be hidden, and remove the class if you want the element to be visible.